project/build.scala

```diff
@@ -107,8 +110,8 @@ object Geoprocessing extends Build {
   lazy val summarySettings =
     Seq(
       libraryDependencies ++= Seq(
-        "com.azavea.geotrellis" %% "geotrellis-spark" % Version.geotrellis,
-        "com.azavea.geotrellis" %% "geotrellis-s3" % Version.geotrellis,
+        "com.azavea.geotrellis" %% "geotrellis-spark" % Version.geotrellisOld,
+        "com.azavea.geotrellis" %% "geotrellis-s3" % Version.geotrellisOld,
```

Comment:
Interesting that we can use both GT versions simultaneously. I guess that's a benefit of having a multi-project build.